src/Model/ModelManagerInterface.php

```diff
@@ -220,8 +236,6 @@ public function executeQuery(object $query);
 
     /**
      * @param string[] $fields
-     *
-     * @return SourceIteratorInterface<object>
```

Comment:
Weird ?

Reply:
Psalm complains about it, it was added on master